uk_bin_collection/uk_bin_collection/councils/SouthHollandDistrictCouncil.py

```
@@ -22,115 +12,38 @@ class CouncilClass(AbstractGetBinDataClass):
-                        continue
+        URI = "https://www.sholland.gov.uk/apiserver/ajaxlibrary"
 
-                    text = next_coll_div.get_text(" ", strip=True)
+        data = {
+            "id": 1,
+            "jsonrpc": "2.0",
+            "method": "SouthHolland.Waste.getCollectionDates",
+            "params": {"UPRN": user_uprn}
+        }
+        # Make the GET request
+        response = requests.post(URI, json=data)
```

Ruff (0.14.7)

Probable use of `requests` call without timeout

(S113)